x-pack/plugins/cases/public/common/lib/kibana/hooks.ts

```diff
@@ -167,9 +167,15 @@ export const useNavigation = (appId: string) => {
 export const useApplicationCapabilities = (): { crud: boolean; read: boolean } => {
   const capabilities = useKibana().services.application.capabilities;
   const casesCapabilities = capabilities[FEATURE_ID];
-
   return {
     crud: !!casesCapabilities?.crud_cases,
     read: !!casesCapabilities?.read_cases,
   };
 };
+export const useKibanaCapabilities = (): { visualize: boolean } => {
```

Comment:
I know is unrelated to this PR but do you mind if we merge `useApplicationCapabilities` and `useKibanaCapabilities` to one hook called `useApplicationCapabilities` like:

```
interface Access {
  crud: boolean; 
  read: boolean;
}

export const useApplicationCapabilities = (): { generalCases: Access, visualize: Access} => {
  const capabilities = useKibana().services.application.capabilities;
  const generalCasesCapabilities = capabilities[FEATURE_ID];
  
  return {
      cases: {
          crud: !!casesCapabilities?.crud_cases,
          read: !!casesCapabilities?.read_cases,
      },
      visualize: {
          crud: !!capabilities?.visualize?.save,
          read: !!capabilities?.visualize?.show,
      }
  }
}
```

I have a PR coming that will add another one for `actions`.

Reply:
We decided offline that I will do it in my PR.